services/apps/alcs/src/common/automapper/application-decision-v1.automapper.profile.ts

```diff
@@ -160,6 +160,10 @@ export class ApplicationDecisionProfile extends AutomapperProfile {
         mapper,
         ApplicationDecisionMeeting,
         ApplicationDecisionMeetingDto,
+        forMember(
+          (a) => a.date,
+          mapFrom((ad) => ad.date.getTime()),
```

Comment:
Is this for changing db datetime to be readable by frontend

Reply:
It was sending the date as a string, but we should send dates as numbers to be consistent with the rest of the app.